src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -811,12 +823,48 @@ def progress_patch(self, *args, **kwargs):
 
     def run(self):
         """ starts the REPL """
-        from .progress import ShellProgressView
-        self.cli_ctx.get_progress_controller().init_progress(ShellProgressView())
-        self.cli_ctx.get_progress_controller = self.progress_patch
 
+        # load command table
         self.command_table_thread = LoadCommandTableThread(self.restart_completer, self)
         self.command_table_thread.start()
+        self.command_table_thread.start_time = time.time()
+        print_styled_text([(Style.ACTION, "Loading command table... Expected time around 1 minute.")])
+        progress_bar = IndeterminateProgressBar(cli_ctx=self.cli_ctx, message="Loading command table")
+        # initialize some variables
+        # whether the customer chooses to continue loading
+        continue_loading = False
+        time_spent_on_loading = 0
```

Comment:
```suggestion
```

Reply:
The temporary variables in a loop do not need to be defined repeatedly outside the loop